clicommand/agent_start.go

```diff
@@ -213,6 +214,10 @@ func (asc AgentStartConfig) Features(ctx context.Context) []string {
 		features = append(features, "allowed-repositories")
 	}
 
+	if len(asc.AllowedEnvironmentVariables) > 0 {
+		features = append(features, "allowed-environment-variables")
+	}
+
```

Comment:
This requires some changes to the backend, so it won't work for now. It's only used internally so shouldn't be a blocker to you, so we recommend deleting it.
```suggestion
```